rollup.config.js

```diff
@@ -36,72 +37,85 @@ const postcssPlugins = [
   postcssReporter(),
 ];
 
-export default [
+const basePlugins = [
+  babel({
+    exclude: ['node_modules/**'],
+    runtimeHelpers: true,
+    ...babelOptions(),
+  }),
+  builtins(),
+  cleanup(),
+  commonjs({
+    include: 'node_modules/**',
+    namedExports: {
+      'node_modules/react/index.js': [
+        'Children',
+        'Component',
+        'createElement',
+        'isValidElement',
+      ],
+      'node_modules/react-is/index.js': ['isValidElementType'],
+      'node_modules/flatpickr/dist/l10n/de.js': ['German'],
+    },
+  }),
+  json(),
+  peerDepsExternal(),
+  // this is used for importing our css modules
+  postcss({
+    include: ['**/*.mod.css'],
+    exclude: ['node_modules/**/*.css'],
+    modules: true,
+    importLoaders: 1,
+    localIdentName: '[name]__[local]___[hash:base64:5]',
+    plugins: postcssPlugins,
+  }),
+  // this is used for importing both vendor css (from node_modules) and for
+  // importing our global css which uses tokens that require postcss plugins
+  // (such as color-mod) to be compiled properly.
+  postcss({
+    exclude: ['**/*.mod.css'],
+    include: ['**/*.css'],
+    plugins: postcssPlugins,
+  }),
+  replace({
+    'process.env.NODE_ENV': JSON.stringify('production'),
+  }),
+  resolve(),
+  svgrPlugin({
+    include: ['**/*.react.svg'],
```

Comment:
Maybe a comment for others that this is our way of targeting svgs which mean to become react components?